### ISSUE TITLE
Strikethrough not announced in Word 2016 with NVDA+f or announce formatting changes

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#5800]
Strikethrough is not announced when reading text formatting with NVDA+f in Word 2016.  It is also not identified when "Announce formatting changes after the cursor" (and report font attributes) are turned on.

Strikethrough is announced in WordPad.

**Comment by nvaccessAuto** [comment]
Incubated in 09eab9b58db37f841b29ad3c0c80332772e1172e.
